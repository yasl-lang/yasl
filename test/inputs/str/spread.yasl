echo 'ABCD'->spread()
echo ('ABCD'->spread())
echo collections.list('12345'->spread())
