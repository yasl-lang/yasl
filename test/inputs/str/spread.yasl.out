A, B, C, D
A
[1, 2, 3, 4, 5]
